string                              id
string                              agent_name
uint32                              bid
string                              role
mac_ros_bridge/Item[]               items
mapc_rhbp_ettlinger/AssembleRequest request
uint32                              expected_steps